### DISCUSSION TITLE
Does custom property [XLColumn(Header="Field Name")] supported in .net6+

### DISCUSSION THREAD

**Discussion by windlight1** [ClosedXML/ClosedXML#2230] [Q&A]
I am confused about this document(https://github.com/ClosedXML/ClosedXML/wiki/Inserting-Tables). It said that [XLColumn(Header="Field Name")] is only available in .Net 4, but I checked the source code and tried it in .net6, and it can work

**Comment by igitur** [comment]
The documentation is a bit obsolete. What is meant, is that it's not available for the old NET3.5 builds.

**Reply by windlight1** [reply]
Thank you. 
And could I use this "Header" parameter in .net6+, and will it be kept in the future?

**Reply by igitur** [reply]
Try it and let me know if it works on net60.

**Reply by windlight1** [reply]
I try version 0.102.1, and it works on .net6.
